docker/install.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 LEONARDO_DIR=$1
 cd $LEONARDO_DIR
+rm leonardo*.jar
```

Comment:
Without this, multiple jars can accumulate.  We expect a single jar, and docker `COPY` also fails when there are multiple.

Reply:
`rm -f`? To avoid errors if the jar doesn't exist

Reply:
yes, good call